modules/persistent-storage-csi-manila-dynamic-provisioning.adoc

```diff
@@ -5,15 +5,20 @@
 [id="persistent-storage-csi-manila-dynamic-provisioning_{context}"]
 = Dynamically provisioning Manila CSI volumes
 
-{product-title} installs a StorageClass for each available Manila share type.
+{product-title} installs a storage class for each available Manila share type.
 
-The YAML files that are created are completely decoupled from Manila and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy Pods with applications that safely consume the storage using YAML manifests. You can also provision other access modes, such as ReadWriteOnce (RWO).
+The YAML files that are created are completely decoupled from Manila and from its Container Storage Interface (CSI) plug-in. As an application developer, you can dynamically provision ReadWriteMany (RWX) storage and deploy pods with applications that safely consume the storage using YAML manifests. You can also provision other access modes, such as ReadWriteOnce (RWO).
```

Comment:
Is the storage consumed or used?

> ! consume v
Do not use when a simpler term such as “use” would be more appropriate. Use only to mean “use up.”

Reply:
Either would work but because I also have "using" in the same sentence, I'll stick with "consume" here. Because it's storage, it is being "used up."